src/WebJobs.Extensions.CosmosDB/Trigger/CosmosDBTriggerListener.cs

```diff
@@ -117,15 +122,39 @@ public async Task StopAsync(CancellationToken cancellationToken)
             }
         }
 
-        private void InitializeHost()
+        internal virtual async Task StartProcessorAsync()
         {
             if (this._host == null)
             {
-                this._host = new ChangeFeedEventHost(this._hostName,
-                    this._monitorCollection,
-                    this._leaseCollection,
-                    this._changeFeedOptions,
-                    this._leaseHostOptions);
+                this._host = await this._hostBuilder.BuildAsync().ConfigureAwait(false);
+            }
+
+            await this._host.StartAsync().ConfigureAwait(false);
+        }
+
+        internal virtual void WithHealthMonitor(IHealthMonitor healthMonitor)
+        {
+            this._healthMonitor = healthMonitor ?? throw new ArgumentNullException(nameof(healthMonitor));
+        }
```

Comment:
Is this used anywhere?

Reply:
Good catch, I think I left this accidentally.